tock/tock/views.py

```diff
@@ -33,50 +33,18 @@ def logout(request):
     else:
         return render(request, 'logout.html')
 
+def handler400(request, exception):
+    return render(request, '400.html', {})
```

Comment:
Worth noting: we don't actually have a 400.html, so we're not really testing this properly right now. I'm going to log a separate issue to sort that out.